services/src/bin/main.rs

```diff
@@ -112,16 +117,18 @@ where
     Ok(opentelemetry)
 }
 
-fn console_layer<S>() -> impl Layer<S>
+fn console_layer_with_filter<S, F: Filter<S> + 'static>(filter: F) -> impl Layer<S>
 where
     S: Subscriber,
     for<'a> S: LookupSpan<'a>,
 {
     tracing_subscriber::fmt::layer()
+        .pretty()
```

Comment:
What are the implications of pretty instead of the current tracing?
Doesn't this emit multi-line logs such that there is no one-log-per-file anymore?

Reply:
yes it emits really nice to read multi line output. Since this is only the console layer i see no reason to limit us to single lines. The format of the log file is still singel line.